ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml

```diff
@@ -75,6 +89,12 @@ tests:
       CHECK_MOD_LIST: "false"
     test:
     - ref: go-verify-deps
+- as: verify-requirements
+  commands: |
+    git diff --exit-code
```

Comment:
So, `git` will be able to catch the changes since `get-requirements` image is built from the `src` which would have cloned the repo with the modifications done through `build-requirements`, right?

Reply:
Yes. The pj-rehearse run for the CI job shows the diff for the build requirements files: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/64647/rehearse-64647-pull-ci-openshift-ansible-operator-plugins-main-verify-requirements/1924841604464513024

As there are newer versions available for some of the python packages in the build requirements files these are being used which is causing the diff.